inspirehep/modules/authors/forms.py

```diff
@@ -274,13 +274,26 @@ class AuthorUpdateForm(INSPIREForm):
 
     """Author update form."""
 
-    # Hidden field to hold author id information
     bai = fields.StringField(
-        widget=HiddenInput()
```

Comment:
Is there any benefit on changing this? Removing the `HiddenInput ` and then adding it in the `__init__` VS how it was before, hidden by default and then added.